web/telemetry/publisher.go

```diff
@@ -40,7 +40,7 @@ func (tp *TelemetryPublisher) Publish(telemetryName string, installationID uuid.
 	return nil
 }
 
-var apiHost = "https://telemetry.trento-project.io" // this service still does not exist
+var apiHost = "https://telemetry.trento.suse.com" // this service still does not exist
```

Comment:
Hey, @dottorblaster ,
I would remove the comment besides the new domain, as now it will exists hehe

Reply:
By the way, now I notice, `apiHost` looks a quite inaccurate name for this, doesn't it? XD

Reply:
Bad naming here is my fault :sweat_smile: 
Feel free to change or not to something like `telemetryApiHost` `telemetryEndpoint` or whatever makes sense.

Reply:
Comment removed, thanks for that! `apiHost` was quite clear to me but feel free to suggest something else, I think this line is going to be edited a lot to allow configuration via build flags in the future so... :smile:

Reply:
👍🏼 for renaming. I suggest `telemetryServiceUrl`.